draft-todo-yourname-protocol.md

```diff
@@ -24,10 +24,16 @@ abbrev: "TODO - Abbreviation"
 category: info
 
 docname: draft-todo-yourname-protocol-latest
+submissiontype: IETF
+number:
+date:
+consensus: true
 v: 3
 area: AREA
 workgroup: WG Working Group
-keyword: Internet-Draft
+keyword:
+ - TODO1
+ - TODO2
```

Comment:
```suggestion
 - blockchain
```

Reply:
It's only a blockchain if it comes from the blockchain region of France. Otherwise it's...